### ISSUE TITLE
Wrong results on ML1M 

### ISSUE THREAD

**Issue by xiaxin1998** [agiresearch/OpenP5#37]
The result is like this:
The average training loss for epoch 10 is 0.6044511198997498
Save the current model to ../model/ML1M/1_1_1_1_20_1991_ML1M_sequential,straightforward_t5-small_collaborative_0.001_10_64_3,3_prompt.pt
Load model from ../model/ML1M/1_1_1_1_20_1991_ML1M_sequential,straightforward_t5-small_collaborative_0.001_10_64_3,3_prompt.pt
testing ML1M dataset on sequential task
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 151/151 [01:37<00:00,  1.55it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 151/151 [01:40<00:00,  1.50it/s]
hit@5: 0.16440397350993377
hit@10: 0.24850993377483444
ndcg@5: 0.10880534719399257
ndcg@10: 0.13588957776424868
testing ML1M dataset on straightforward task
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 151/151 [01:20<00:00,  1.88it/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 151/151 [01:22<00:00,  1.84it/s]
hit@5: 0.008940397350993378
hit@10: 0.022019867549668875
ndcg@5: 0.004666659666548987
ndcg@10: 0.008816655903199199

I use the same command in the repo. can you give me a hint to get the reported results? Thanks

**Comment by shuyuan-x** [comment]
Please refer to our test command. We reported the results with filtered test data, which filters out the samples that already appeared in the training data.
